tests/test_gradient.py

```diff
@@ -9,53 +9,52 @@
 
 class TestGradient(object):
 
-    @pytest.mark.parametrize('space_order', [4])
-    @pytest.mark.parametrize('kernel', ['OT2'])
-    @pytest.mark.parametrize('shape', [(70, 80)])
-    def test_gradient_checkpointing(self, shape, kernel, space_order):
+    @pytest.mark.parametrize('dtype', [np.float32, np.float64])
+    @pytest.mark.parametrize('opt', [('advanced',
+                                      {'openmp': True, 'par-dynamic-work': 0}),
```

Comment:
I still don't understand why do we keep messing around with `par-dynamic-work` ? is it because we rather want to _enforce_ it?

Reply:
Dropped.